scheds/rust/scx_layered/src/main.rs

```diff
@@ -2147,6 +2155,179 @@ impl<'a> Scheduler<'a> {
         weighted
     }
 
+    fn compute_target_llcs(target: usize, topo: &Topology) -> (usize, usize) {
```

Comment:
Maybe add some high level comments on what the function is doing and why?

Reply:
Fixed.